README.md

```diff
@@ -2,166 +2,72 @@
-31, 2019. http://arxiv.org/abs/1906.00091.
+### Docker Containers that include Merlin Models
 
-![img.png](docs/img/dlrm.png)
+Merlin Models is pre-installed in the NVIDIA Merlin Docker containers that are available in the [NVIDIA container repository](https://ngc.nvidia.com/catalog/containers/nvidia:merlin) in three different containers:
 
-High-level API:
+<!-- prettier-ignore-start -->
 
-```python
-import merlin_models.tf as ml
-
-dlrm = ml.DLRMBlock(
-    schema,
-    embedding_dim=32,
-    bottom_block=ml.MLPBlock([512, 128]),
-    top_block=ml.MLPBlock([512, 128])
-)
-model = dlrm.connect(ml.BinaryClassificationTask(schema))
-```
+| Container Name             | Container Location | Functionality |
+| -------------------------- | ------------------ | ------------- |
+| merlin-tensorflow-training | [https://ngc.nvidia.com/catalog/containers/nvidia:merlin:merlin-tensorflow-training](https://ngc.nvidia.com/catalog/containers/nvidia:merlin:merlin-tensorflow-training) | Transformers4Rec, NVTabular, TensorFlow, and HugeCTR Tensorflow Embedding plugin |
+| merlin-pytorch-training    | [https://ngc.nvidia.com/catalog/containers/nvidia:merlin:merlin-pytorch-training](https://ngc.nvidia.com/catalog/containers/nvidia:merlin:merlin-pytorch-training)    | Transformers4Rec, NVTabular and PyTorch
+| merlin-inference           | [https://ngc.nvidia.com/catalog/containers/nvidia:merlin:merlin-inference](https://ngc.nvidia.com/catalog/containers/nvidia:merlin:merlin-inference)           | Transformers4Rec, NVTabular, PyTorch, and Triton Inference |  |
 
-Low-level API:
+<!-- prettier-ignore-end -->
 
-```python
-import merlin_models.tf as ml
+To use these Docker containers, you'll first need to install the [NVIDIA Container Toolkit](https://github.com/NVIDIA/nvidia-docker) to provide GPU support for Docker. You can use the NGC links referenced in the table above to obtain more information about how to launch and run these containers.
 
-dlrm_inputs = ml.ContinuousEmbedding(
-    ml.InputBlock(schema, embedding_dim_default=128),
-    embedding_block=ml.MLPBlock([512, 128]),
-    aggregation="stack"
-)
-dlrm = dlrm_inputs.apply(ml.DotProductInteraction(), ml.MLPBlock([512, 128]))
-  ```
+<!-- Need core benefits, Common use cases, or Highlights -->
 
-### DCN-V2
+## Core Features
 
-Wang, Ruoxi, Rakesh Shivanna, Derek Z. Cheng, Sagar Jain, Dong Lin, Lichan Hong, and Ed H. Chi. “DCN V2: Improved Deep &
-Cross Network and Practical Lessons for Web-Scale Learning to Rank Systems.” ArXiv:2008.13535 [Cs, Stat], October 20, 2020. http://arxiv.org/abs/2008.13535.
+To learn about the core features of Merlin Models, see the [Retrieval Models](docs/source/retrieval_models.md) page.
+The key retrieval models are as follows:
 
-![img.png](docs/img/dcn-v2.png)
+* Matrix factorization
+* YouTube DNN retrieval
+* Two tower
+* Ranking models such as DLRM and DCN-V2
```

Comment:
These are `Ranking Models`. For some background for this, could be good to watch [this](https://www.youtube.com/watch?v=5qjiY-kLwFY) talk from Even.

Reply:
Good video.  At this point though, I'm not convinced that it's helpful to treat "ranking models" as a proper noun.  There's also some precedent for using plain lower-case text for other model types in the terminology repository that the docs team uses.